database/migrations/2018_03_31_104532_create_media_table.php

```diff
@@ -16,6 +16,8 @@ public function up()
             $table->string('name')->unique();
             $table->boolean('is_public')->default(false);
             $table->integer('owner_id')->nullable();
+            $table->string('preview_name');
```

Comment:
let's make preview as a separate media object and store it's id in `preview_id` field

Reply:
Done.